components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/builders/SAMLArtifactBuilder.java

```diff
@@ -103,13 +107,13 @@ public String buildSAML2Artifact(SAMLSSOAuthnReqDTO authnReqDTO, String sessionI
             Assertion assertion = persistAssertion(authnReqDTO, initTimestamp, sessionIndexId);
             assertionId = assertion.getID();
         }
-        persistSAML2ArtifactInfo(sourceID, messageHandler, authnReqDTO, sessionIndexId, initTimestamp,
+        persistSAML2ArtifactInfo(sourceIDString, messageHandlerString, authnReqDTO, sessionIndexId, initTimestamp,
                 expTimestamp, assertionId);
 
         return Base64.encode(artifactByteArray);
     }
 
-    private void persistSAML2ArtifactInfo(byte[] sourceID, byte[] messageHandler, SAMLSSOAuthnReqDTO authnReqDTO,
+    private void persistSAML2ArtifactInfo(String sourceID, String messageHandler, SAMLSSOAuthnReqDTO authnReqDTO,
```

Comment:
name 'sourceID' is inconsistent. it's used as 'sourceId' at other places. make it consistant